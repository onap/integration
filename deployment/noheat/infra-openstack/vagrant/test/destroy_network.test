#!/bin/sh

export NETWORK_NAME='onap_ci_lab'

export VAGRANT_CWD='..'

set_up() {
    vagrant up
    vagrant provision --provision-with=run_playbook_create
    vagrant provision --provision-with=run_playbook_destroy
}

check() {
    local net="$1"
    vagrant ssh operator --no-tty -c \
        "export OS_CLOUD=openstack; openstack network list -fcsv" \
        | grep "$net" \
        || echo "Network ${net} not found."
}

set_up >/dev/null # drop provisioning output
check "$NETWORK_NAME"
