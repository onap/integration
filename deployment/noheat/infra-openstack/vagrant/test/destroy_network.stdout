Network onap_ci_lab not found.
